spec/models/inventory_food_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe InventoryFood, type: :model do
+  let(:user) { User.new(name: 'pepe', email: 'email1@email.com', password: 'abcdef') }
+  @inventory = Inventory.new(user_id: @user)
+  @food = Food.new(
+    name: 'Rice',
+    measurement_unit: 'kg',
+    price: 5.99
```

Comment:
![image](https://github.com/katarighe/recipe-app/assets/65068771/56dd63a0-cd92-4e28-bcda-8937b02e09dc)

- [ ] Just like previous reviewers said, I think you  need to solve your tests, this error is happing because you are using a table field that doesnot exist in the Food table.
```suggestion
    price_dollars: 5.99
```